lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -18,6 +19,7 @@ object EstimationHandler extends CohortHandler {
 
   // TODO: move to config
   private val batchSize = 150
+  private val priceCappingMultiplier = 1.2
```

Comment:
Should move this to a common location to avoid it changing between estimation and amendment.

Reply:
Yes. I intend to move both in a PR shortly.